docs/getting-started/integration-toolkit.md

```diff
@@ -13,59 +13,7 @@ in integration and interoperability projects. The Toolkit is both a recognized
 The Toolkit provides a suite of software tools and documentation to help users
 design, build, and automate integrations.
 
-## Lightning Coming Soon!
```

Comment:
I do not think you should remove all mentions of `Lightning` on this page... if you do, Lightning does not show up anywhere here and is not on the diagram screenshot on this page. @amberrignell 

So either (1) update the Toolkit diagram to include Lightning, and/or (2) keep a small section mentioning Lightning as the latest and greatest extension of the DPG... and then maybe link to your main Lightning page.